pkg/noderesourcetopology/score_test.go

```diff
@@ -133,7 +144,7 @@ func TestNodeResourceScorePlugin(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		nodesMap, lister := initTest(defaultNUMANodes(topologyv1alpha2.SingleNUMANodeContainerLevel))
+		nodesMap, lister := initTest(defaultNUMANodes(topologyv1alpha2.SingleNUMANodeContainerLevel), nrtPassthrough)
```

Comment:
I believe more option would introduce in the future.
Hence, I would suggest to use functional options: https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis

Reply:
The comment is not in place, I was referring to `defaultNUMANodes` function